packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -12,7 +12,6 @@
  * @prop --calcite-panel-footer-space: Specifies the spacing of the component's footer.
  * @prop --calcite-panel-header-background-color: Specifies the component header's background color.
  * @prop --calcite-panel-header-border-block-end: [Deprecated] No longer necessary. Specifies the component header's block end border.
- * @prop --calcite-panel-header-color: [Deprecated] Use `--calcite-panel-heading-text-color` instead. Specifies the component header's  color.
```

Comment:
This actually wasn't a prop that was there before so we can just remove it. Should have been removed.